presentation/frontend/src/test/Presentation.test.ts

```diff
@@ -120,6 +120,22 @@ describe("Presentation", () => {
       expect(Presentation.presentation.activeLocale).to.eq("other");
       expect(Presentation.selection.scopes.activeLocale).to.eq("other");
     });
+
+    it("sets `startIModelInitialization` and `ensureIModelInitialized` callbacks of `PresentationManager` as proxies to `FavoritePropertiesManager`", async () => {
+      await Presentation.initialize({ presentation: { activeLocale: "test" } });
+      const manager = Presentation.presentation;
+      const favoriteProperties = Presentation.favoriteProperties;
+
+      const startInitializationSpy = sinon.spy(favoriteProperties, "startConnectionInitialization");
+      const ensureInitializedSpy = sinon.spy(favoriteProperties, "ensureInitialized");
+      const imodel = moq.Mock.ofType<IModelConnection>();
```

Comment:
We're trying to move away from `typemoq`. I believe you don't really need the imodel here, so you should be good with just `const imodel = {} as unknown as IModelConnection`.